com.jetbrains.RustRover.yaml

```diff
@@ -12,21 +12,24 @@ finish-args:
   - --share=network
   - --socket=pulseaudio
   - --filesystem=host
+  - --talk-name=org.freedesktop.Flatpak
   - --talk-name=org.freedesktop.Notifications
   - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.gnome.keyring.SystemPrompter
```

Comment:
Why?

Reply:
I think it's needed when using git with GPG. But since I haven't been able to get it to work with the pinentry module, I removed it from the manifest.